app/services/facets/entry.rb

```diff
@@ -80,6 +80,10 @@ def template_compatibility_properties(*property_symbols)
       @compatibility_properties = property_symbols
     end
 
+    def load_api_controller
+      @api_controller.to_s.constantize
```

Comment:
Note, this was inspired by https://github.com/Apipie/apipie-rails/blob/master/lib/apipie/application.rb#L413